.github/PULL_REQUEST_TEMPLATE/pull_request_template_berk.md

```diff
@@ -0,0 +1,86 @@
+# Guidelines
+
+## _Soft_ Schema Freeze
+
+The `nmdc-schema` and `berkeley-schema-fy24` schemas are under a soft freeze, which means changes **should not** be made that have any downstream implications. To ensure this, all PRs created creating during the freeze will be closely reviewed with **every** component of the NMDC system in mind.
+
+## Reviewers
+
+To ensure no changes are made unexpectedly, PR creators will request reviews from _all_ [Berkeley Schema Roll Out task coordinators](https://docs.google.com/document/d/1XXN1YuaBuSkxPXeiLKm5YxYzXTamBPQrzzeLhlh7PWs/edit#heading=h.u52g8v319adh).
+
+We expect task coordinators to review PRs and provide feedback/approval within 1 week of when they are identified as reviewers. 
+
+PRs will **NOT** be merged until all task coordinators (or one of their delegates) have approved. 
+
+Expedition, questions, and discussion can happen at any meeting.
+
+Delays in review & mergeing should be address in meetings or with NMDC Leadership.
+
+# PR Information
+
+## What type of PR is this? (check all applicable)
+
+- [ ] Refactor
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Optimization
+- [ ] Documentation
+- [ ] Schema change: Content
+  - slot or class name
+  - slot multiplicity changes (from a single value to a list or vice versa)
+  - any changes to the ways slots are assigned to any classes
+  - creating a new slot/Class
+  - Enum value changes
+- [ ] Schema change: Cleanup
+  - descriptions of slots, classes, or enums
+  - removal of unused, commented, or invalid code
+  - updated mappings of terms, classes, or slots to ontologies
+  - added an additional Enum to accommodate new or future metadata
```

Comment:
FYI: I'm working on a proposal for these two bullet points (and their contents). I'll post it here once I'm done.

Reply:
```suggestion
- [ ] Schema change — Structure and content; for example:
  - adds or renames a class, slot, or Enum
  - changes the multiplicity of a slot (e.g. string ↔ list)
  - changes how a slot is used by a class
  - adds or updates an Enum value
- [ ] Schema change — Cleanup and documentation; for example:
  - updates the description of a class, slot, or Enum
  - updates comments
  - deletes unused code
```

Reply:
I'll close this proposal since the code is was based on has been changed via a different means. I will re-express the proposal in a few minutes.